docs/source/dataframe-create.rst

```diff
@@ -57,23 +59,61 @@ Pandas:
 Creating
 --------
 
-Reading from various locations
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Read from CSV
+~~~~~~~~~~~~~
 
-For text, CSV, and Apache Parquet formats, data can come from local disk,
-the Hadoop File System, S3FS, or other sources, by prepending the filenames with
-a protocol:
+You can use :func:`read_csv` to read one or more CSV files into a Dask DataFrame.
+It supports loading multiple files at once:
+  
+.. code-block:: python
+
+   >>> df = dd.read_csv('myfiles.*.csv')
+
+Or you can break up a single large file with the ``blocksize`` parameter:
```

Comment:
"Break up" here refers to loading a single large file into multiple partitions, but given the context from the previous example about loading multiple files, this implies (to me, anyways) that using `blocksize` will result in multiple _files_ on disk. This is clarified in the following paragraph, so maybe nothing needs to be done, but I thought I'd mention it in case it was easy to rearrange.